hardware/asic/skywater/skywater130.mk

```diff
@@ -1 +1,7 @@
 #Makefile for OpenLane and OpenRAM started on 2/11/21
```

Comment:
rename this file and call it simply Makefile

Reply:
Ok Sir